Formula/bwidget.rb

```diff
@@ -0,0 +1,30 @@
+class Bwidget < Formula
+  desc "Tcl/Tk script-only set of megawidgets to provide the developer additional tools"
+  homepage "https://core.tcl-lang.org/bwidget/home"
+  url "https://downloads.sourceforge.net/project/tcllib/BWidget/1.9.16/bwidget-1.9.16.tar.gz"
+  sha256 "bfe0036374b84293d23620a7f6dda86571813d0c7adfed983c1f337e5ce81ae0"
+  license "TCL"
+
+  uses_from_macos "tcl-tk" => :build
+
+  def install
+    cp_r ".", "#{share}/"
```

Comment:
```suggestion
    lib/"bwidget".install Dir["*"]
```

Reply:
Apologies for the syntax question. This fails with `NoMethodError: undefined method `install' for "bwidget":String`.

There's no actual installer here -- the method as I understand it is just to copy all the *.tcl files to lib, and the lang and images directories over subdirectories of lib.

I tried using the following, but then the images are simply copied to the top-level and the lang appears to have nothing happen. Per https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-cp_r I assume this should be working but I'm clearly doing this wrong.
```
cp_r Dir.glob("*.tcl"), "#{lib}"
cp_r "images", "#{lib}"
cp_r "lang", "#{lib}"
```

As seen above, `cp_r ".", "#{lib}"` does work without modifying `tcl-tk`, with the unpleasant side effect of adding the readme and changelog.

Reply:
> Apologies for the syntax question. This fails with `NoMethodError: undefined method 'install' for "bwidget":String`.

Sorry, I left out a pair of parentheses. That should've been:
```
    (lib/"bwidget").install Dir["*"]
```

> There's no actual installer here -- the method as I understand it is just to copy all the *.tcl files to lib, and the lang and images directories over subdirectories of lib.

That's what the `install` method on the `lib/"widget"` pathname does.

Reply:
Change made with 7323f19.